dev/com.ibm.ws.jpa.tests.eclipselink.query_jpa_3.2_fat/build.gradle

```diff
@@ -66,6 +72,18 @@ task copyCommonFiles {
       into new File('publish')
     }
   }
+  
+    dependsOn ':com.ibm.ws.jpa.tests.eclipselink.query_32_fat.common:build'
+  doLast {
+    copy {
+      from project(':com.ibm.ws.jpa.tests.eclipselink.query_32_fat.common').file('test-applications')
+      into new File('test-applications')
+    }
+    copy {
```

Comment:
remove the copy of "publish"; not needed.  all servers are already provided by common and 31_common